x-pack/plugin/wildcard/src/test/java/org/elasticsearch/xpack/wildcard/mapper/WildcardFieldMapperTests.java

```diff
@@ -179,6 +179,41 @@ public void testTooBigQueryField() throws IOException {
         reader.close();
         dir.close();
     }
+    
+    
+    public void testTermQueryIgnoresWildcardSyntax() throws IOException {
+        // Test for bug https://github.com/elastic/elasticsearch/issues/62081
+        // where term queries honoured wildcard quwery syntax
```

Comment:
I don't think we need the link on the issue. The test name is descriptive enough imo.